lucky-number-model/Models/LuckyNumber.cs

```diff
@@ -8,14 +8,85 @@ namespace lucky_number_model.Models
 {
     public class LuckyNumber
     {
+        private int _balance;
+        private bool _isWinner = false;
+        private int[] _spinner = new int[3];
+        
+
         [Required]
         [Display(Name = "Lucky Number")]
         [Range(1, 9, ErrorMessage = "Number must be between 1 and 9")]
         public int Number { get; set; } // the user's choice of a lucky number
 
         [Required]
-        public decimal Balance { get; set; } // the balance for this game
+        public int Balance {
+            get
+            {
+                if (isWinner)
```

Comment:
Generally only use the instance variables NOT the properties within the Model, e.g.
if(_isWinner)